lib/components/ms-input/MsAddressInput.vue

```diff
@@ -54,29 +62,8 @@ function setValue(value: string): void {
 
 async function doQuery(query: string): Promise<void> {
   querying = true;
-  const addressesFound = await geoapifyApi.autocomplete(query);
-  if (addressesFound.length === 0) {
-    return;
-  }
-  const popover = await popoverController.create({
-    component: MsAddressDropdown,
-    cssClass: 'address-dropdown-popover',
-    componentProps: {
-      // Limit ourselves to the first five
-      addresses: addressesFound.slice(0, 5),
-    },
-    trigger: 'ms-address-input',
-    side: 'bottom',
-    alignment: 'start',
-    showBackdrop: false,
-  });
-  await popover.present();
-  const { role, data } = await popover.onDidDismiss();
-  await popover.dismiss();
-  if (role === MsModalResult.Confirm && data) {
-    querying = false;
-    emits('addressSelected', data.address as Address);
-  }
+  addressesFound.value = await geoapifyApi.autocomplete(query);
```

Comment:
Slice it in one line or use a temporary variable, we don't want `addressesFound` to have a temporary value.